lua/lspconfig/idris2.lua

```diff
@@ -0,0 +1,32 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.idris2 = {
+  default_config = {
+    cmd = { 'idris2-lsp' },
+    filetypes = { 'idris2' },
+    root_dir = util.root_pattern '*.ipkg',
+  },
+  docs = {
+    description = [[
+https://github.com/idris-community/idris2-lsp
+
+Idris 2 language server.
+
+Note that this requires a build of Idris 2 that includes the Idris 2 API
+package. For instructions, see
+<https://github.com/idris-lang/Idris2/blob/main/INSTALL.md>. Steps 5 and 8 are
+listed as "optional" in that guide, but they are required for this build.
+
+Note also that you need to install a version of Idris2-Lsp that is compatible
+with your version of Idris 2. There should be a branch corresponding to every
+released Idris 2 version after v0.4.0. Use the latest commit on that branch. If
+your Idris 2 version is newer than the newest Idris2-Lsp branch, use the latest
+commit on the `master` branch, and set a reminder to check the Idris2-Lsp repo
+for the release of a compatible versioned branch.
```

Comment:
Is this idris 2 filetype bundled with vim/neovim? If not should mention here.

Reply:
It is not. Added more info.